xgcm/grid.py

```diff
@@ -24,6 +24,9 @@ def _maybe_promote_str_to_list(a):
         return a
 
 
+_VALID_BOUNDARY = [None, "fill", "extend", "extrapolate"]
```

Comment:
I think we might not be supporting `extrapolate` anymore? Ill have to check on that.

Reply:
OK let me know. Out of curiosity, why not?

Reply:
Actually we do [here](https://github.com/xgcm/xgcm/blob/03c165c225545e5bb4aa22eb08ad9a366b4879e3/xgcm/duck_array_ops.py#L100).
Could you parameterize it in the test function so we make sure that its tested?

Reply:
ahaha a lot of tests fail so I marked them as xfail. We should fix them in a separate PR. Mostly it's because the code that creates expected results is not set up to do extrapolate.